src/app/ngrx/effects/castle.effects.spec.ts

```diff
@@ -350,7 +358,8 @@ describe('CastleEffects', () => {
       expect(effects.loadEpisodePage$).toBeObservable(expected);
     });
 
-    it('should load next page of episodes', () => {
+    // Episode loading TBD
```

Comment:
(stop loading episodes page by page and only load episode pages when selected in pager)

Reply:
👍